components/ai_chat/core/browser/engine/engine_consumer.h

```diff
@@ -82,6 +82,8 @@ class EngineConsumer {
   }
 
  protected:
+  bool ShouldDoEarlyCompletion(
```

Comment:
Please add comment as to how the implementer should fill this out

Reply:
added in e7d074eb69fd14bcce4cdb44c83d845d606d9c0d